doc/source/serve/production-guide/kubernetes.md

```diff
@@ -28,18 +28,25 @@ Follow the [KubeRay quickstart guide](kuberay-quickstart) to:
 * Prepare a Kubernetes cluster
 * Deploy a KubeRay operator
 
-(serve-deploy-app-on-kuberay)=
-## Deploying a Serve application
+## RayService CR
+Once the KubeRay controller is running, you can manage your Ray Serve application by creating and updating a `RayService` custom resource (CR, here is [example](https://github.com/ray-project/kuberay/blob/release-0.5/ray-operator/config/samples/ray_v1alpha1_rayservice.yaml)).
+Under the `spec` section in the `RayService` CR, you can find the following fields:
+
+**`serviceUnhealthySecondThreshold`**: represents the threshold in seconds that defines when a service is considered unhealthy. Default is 60 seconds. When the service is unhealthy, the KubeRay Service controller will try to recreate a new cluster and deploy the application to the new cluster.
```

Comment:
```suggestion
**`serviceUnhealthySecondThreshold`**: the number of seconds before an unhealthy service is marked unhealthy. Default is 60 seconds. When the service is marked unhealthy, the KubeRay Service controller creates a new cluster, deploys the Serve application to it, and then switches traffic to the new cluster.
```

Reply:
Could you give more details on what makes a service unhealthy? E.g. at least one deployment is `UNHEALTHY`? I believe the `app_status` is also relevant here.